Lib/mutatorMath/ufo/instance.py

```diff
@@ -279,8 +338,7 @@ def addKerning(self, instanceLocation=None, sources=None):
                 self.logger.exception("Error processing kerning data. %s", items)
                 return
             instanceObject = m.makeInstance(instanceLocation)
-            if self.roundGeometry:
-                instanceObject.round()
+            instanceObject.round() # Kerning values must be integer, so we don't check self.roundGeometry
```

Comment:
This means kerning data is always rounded, regardless of the self.roundGeometry flag? If that's what you have in mind then I don't think this is a good idea. If the instance is to play a role in a further calculation it is a shame the kerning data has been rounded already.